empiric-package/empiric/core/__init__.py

```diff
@@ -1,11 +1,3 @@
 import logging
 
-LOGGER = logging.getLogger(__name__)
-LOGGER.setLevel(logging.DEBUG)
-
-# defines the stream handler
-_ch = logging.StreamHandler()  # creates the handler
-_ch.setLevel(logging.INFO)  # sets the handler info
-
-# adds the handler to the global variable: log
-LOGGER.addHandler(_ch)
+logging.getLogger(__name__).addHandler(logging.NullHandler())
```

Comment:
This seems like the right way to do things - out of curiosity, how would a user of our SDK add their own e.g. `StreamHandler`?

Reply:
i put the logger example in the to utils to show how a user could import that.  But in general, they could just get the logger using `getLogger` and add their own handler.